isc_sick/src/LMS1xx_node.cpp

```diff
@@ -10,6 +10,7 @@
 #include <cstdio>
 #include <LMS1xx/LMS1xx.h>
 #include "ros/ros.h"
+#include <ros/console.h>
```

Comment:
Why aren't the logging messages in this file updated?

Reply:
The logging was already done the way it should be, as I fixed in the other file. I think I added that include, but it may not be necessary since it was already included in LMS1xx.h... Does it work that way?

Reply:
Ohhhhhhhh I got confused about the changes you made. I'll approve all these. You're good.